src/v0/destinations/marketo_bulk_upload/config.js

```diff
@@ -0,0 +1,36 @@
+const ABORTABLE_CODES = ['ENOTFOUND', 'ECONNREFUSED', '601', '603', '605', '609', '610'];
```

Comment:
'ENOTFOUND', 'ECONNREFUSED' we dont need these here, here we are only concered with Marketo application level error code or record level error code

Reply:
done